src/divider.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import StylePropable from './mixins/style-propable';
+import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
+import ThemeManager from './styles/theme-manager';
+
+const Divider = React.createClass({
+
+  mixins: [StylePropable],
+
+  contextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+
+  propTypes: {
+    /**
+     * If true, the divider will be indented by 72px
+     */
+    inset: React.PropTypes.bool,
+
+    /**
+     * Override the inline-styles of the list divider's root element.
+     */
+    style: React.PropTypes.object,
```

Comment:
Yeah, the react docs make it seem like [this](https://facebook.github.io/react/docs/context.html#referencing-context-in-stateless-functional-components) is possible.

> Stateless functional components are also able to reference context if contextTypes is defined as a property of the function. The following code shows the Button component above written as a stateless functional component.

If I understand you correctly, I am also confused as to why context isn't enough on it's own to store muiTheme and we have to use state....